internal/provider/provider.go

```diff
@@ -0,0 +1,32 @@
+package provider
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
```

Comment:
We may need to sort the imports (see https://github.com/hashicorp/terraform-provider-time/pull/114#discussion_r938640440).